packages/main/src/Tokenizer.ts

```diff
@@ -241,31 +250,33 @@ class Tokenizer extends UI5Element {
 		}
 	}
 
-	_delete(event) {
-		if (!event.detail) { // if there are no details, the event is triggered by a click
-			this._tokenClickDelete(event, event.target);
+	_delete(e: CustomEvent<TokenDeleteEventDetails>) {
+		const target = e.target as Token;
+		if (!e.detail) { // if there are no details, the event is triggered by a click
+			this._tokenClickDelete(e, target);
 			return;
 		}
 
 		if (this._selectedTokens.length) {
-			this._selectedTokens.forEach(token => this._tokenKeyboardDelete(event, token));
+			this._selectedTokens.forEach(token => this._tokenKeyboardDelete(e, token));
 		} else {
-			this._tokenKeyboardDelete(event);
+			this._tokenKeyboardDelete(e, target);
 		}
 	}
 
-	_tokenClickDelete(event, token) {
+	_tokenClickDelete(e: CustomEvent, token: Token) {
```

Comment:
If it's the same event as in _delete - add the detail's type: `CustomEvent<TokenDeleteEventDetails>`

Reply:
Done